api/oc_server_api.c

```diff
@@ -697,7 +715,7 @@ handle_separate_response_request(coap_separate_t *request,
     response_state = oc_blockwise_alloc_response_buffer(
       oc_string(request->uri), oc_string_len(request->uri), &request->endpoint,
       request->method, OC_BLOCKWISE_SERVER,
-      (uint32_t)response_buffer->response_length);
+      (uint32_t)response_buffer->response_length, false);
```

Comment:
in some cases you want to generate etag or ?

Reply:
Yes, to keep the previous implementation where the etag was generated for a blockwise message with multiple blocks, otherwise it was ignored.